tests/python/mkl/test_subgraph.py

```diff
@@ -794,6 +810,7 @@ def test_single_fc():
 
 
 @with_seed()
+@unittest.skip('skip for MKL-DNN 1.0 integration: https://github.com/apache/incubator-mxnet/projects/16')
```

Comment:
I assume this is temporary and will be removed before we move forward with the PR.

Reply:
Correct. These tests will be re-enabled once the corresponding functionality is recovered.

Reply:
These tests will be recovered in the follow-up PRs of this integration task before we merge this feature branch back to the master.